python/src/trezorlib/benfen.py

```
@@ -0,0 +1,55 @@
+
+@expect(messages.BenfenAddress, field="address", ret_type=str)
+def get_address(
+    client: "TrezorClient", address_n: "Address", show_display: bool = False
+) -> "MessageType":
+    return client.call(
+        messages.BenfenGetAddress(address_n=address_n, show_display=show_display)
+    )
+
+
+@expect(messages.BenfenSignedTx)
+def sign_tx(client: "TrezorClient", address_n: "Address", raw_tx: bytes,coin_type: bytes ):
```

Ruff (0.8.2)

Missing return type annotation for public function `sign_tx`

(ANN201)